src/components/CivicProfileForms/BasicInfo.jsx

```diff
@@ -1,8 +1,169 @@
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+// Hook Imports
+import { useCivicProfile } from '@hooks';
+// Component Imports
+import { FormSection } from '../Form';
 
-const BasicInfo = () => <Typography>Basic Info</Typography>;
+/**
+ * BasicInfo Component - A form to fill out basic user info
+ *
+ * @memberof CivicProfileForms
+ * @name BasicInfo
+ * @returns {React.JSX.Element} The BasicInfo Component
+ */
+const BasicInfo = () => {
+  const { data, add, isSuccess } = useCivicProfile();
+  const [formData, setFormData] = useState({
+    legalFirstName: '',
+    legalLastName: '',
+    legalDOB: null,
+    legalGender: ''
+  });
+
+  useEffect(() => {
+    if (isSuccess) {
+      setFormData((prevFormData) => ({ ...prevFormData, ...data }));
+    }
+  }, [isSuccess, data]);
+
+  const handleChange = (event) => {
+    if (event && event.target) {
+      const { name, value } = event.target;
+      setFormData((prevFormData) => ({ ...prevFormData, [name]: value }));
+    } else {
+      setFormData({
+        ...formData,
+        legalDOB: event
+      });
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!isSuccess) {
+      return;
+    }
+    add(formData);
+  };
+
+  const handleClear = () => {
+    setFormData({ legalFirstName: '', legalLastName: '', legalDOB: '', legalGender: '' });
+  };
+
+  return (
+    <FormSection
+      title="Basic Information"
+      component="form"
+      onSubmit={handleSubmit}
+      autoComplete="off"
+    >
+      <Grid container spacing={{ xs: 2, sm: 2 }}>
+        <Grid item xs={12} sm={6}>
+          <TextField
+            id="hmis-basic-info-first-name"
+            name="legalFirstName"
+            label="Legal first name"
+            onChange={handleChange}
+            value={formData.legalFirstName}
+            fullWidth
+            autoFocus
+          />
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <TextField
+            id="hmis-basic-info-last-name"
+            name="legalLastName"
+            label="Legal last name"
+            onChange={handleChange}
+            value={formData.legalLastName}
+            fullWidth
+          />
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <FormControl fullWidth>
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="hmis-basic-info-date-of-birth"
+                name="legalDOB"
+                label="Date of birth"
+                onChange={handleChange}
+                value={formData.legalDOB}
+                format="YYYY-MM-DD"
+                type="date"
+                disableFuture
+                openTo="year"
+                views={['year', 'month', 'day']}
+              />
+            </LocalizationProvider>
+            <FormHelperText>YYYY-MM-DD</FormHelperText>
```

Comment:
Not an issue here, but we don't need to obligate this format. As long as we properly convert to this format during storage, we can allow the user to enter whatever format they want.

We can improve this later though.

Reply:
I'll remove line 113. Would it be fine to leave the rest unchanged for now (such as `format="YYYY-MM-DD"`)?

Reply:
If we are going to require that the user submits data as `YYYY-MM-DD`, then we need to keep line 113. We need to communicate what format we require the dates to be submitted it. 

My point is that we are not obligated to receive user input in that format if we think another format is easier for users. As long as we store the date in the format `YYYY-MM-DD`, we can have the user enter the date in whatever format we want.

Reply:
Okay, I'll return the helper text for the time being.